src/document.h

```diff
@@ -496,6 +480,22 @@ struct Document {
         return;
     }
 
+#ifdef __WXGTK__

```

Comment:
just use a `-> auto` return type

Reply:
And put the helper function above that did the trick!
Thanks for your feedback! :-)